src/go/k8s/controllers/redpanda/cluster_controller.go

```diff
@@ -37,14 +37,15 @@ import (
 var (
 	errNonexistentLastObservesState = errors.New("expecting to have statefulset LastObservedState set but it's nil")
 	errNodePortMissing              = errors.New("the node port is missing from the service")
+	errNoneExistingImagePullPolicy  = errors.New("none existing image pull policy")
```

Comment:
typo: "nonexistent" or maybe "invalid"?

(ok to leave for later if you need to merge)